core/editor.js

```diff
@@ -675,7 +675,7 @@
 			if ( element.is( 'textarea' ) )
 				element.setValue( data );
 			else
-				element.setHtml( data );
+				element.setHtml( CKEDITOR.tools.htmlSafeByReview(data, 'getData pulls event data directly from the editor itself') );
```

Comment:
Verify that htmlEncodeOutput is set to true. We can also sanitize before the setHtml since it should be an inert string by then. 

Do things break if we remove the if statement that checks htmlEncodeOutput?

Reply:
Removing htmlEncodeOutput does break some tests. It looks like this function can be used with unencoded html https://sourcegraph.com/github.com/ckeditor/ckeditor4/-/blob/tests/core/editor/editor.js?L214-247. It looks like this is a function used by the developer but we may want to sanitize/legacy convert this function.

Reply:
Legacy converted this function